frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -141,7 +142,34 @@ const Campaigns = () => {
           />
         )
       },
-      width: 'md center',
+      width: 'md',
```

Comment:
Looking at the design master, I think should be `md center` so that the buttons are more evenly spaced out.

Reply:
I tried that but the spacing looked funny. This is md center for both
<img width="1114" alt="Screenshot 2020-11-25 at 1 25 07 PM" src="https://user-images.githubusercontent.com/33819199/100186774-abf02380-2f21-11eb-9eff-a50a488241b8.png">

Reply:
Note the space on the right between delivery report and the duplicate icon

Reply:
<img width="1122" alt="Screenshot 2020-11-25 at 1 33 54 PM" src="https://user-images.githubusercontent.com/33819199/100187344-e3ab9b00-2f22-11eb-99ad-b85e724887d4.png">
how about md center and md?

Reply:
The "Delivery report" button can be shorted to "Report" to make space, as it's pretty clear what the report is about. 

This can also give more space to column spacing, could we try making the 'name' column wider and pushing the other columns over?

For the text button icons, we created a new issue [here](https://github.com/opengovsg/postmangovsg/issues/859)